main.cpp

```diff
@@ -13,75 +13,109 @@ What you create in this project will be used as the basis of Project 5 in the co
-        1)
-        2)
-        3)
-        4)
-    Name 3 things you can do with the:   Instrument Cluster
-        1)
-        2)
-        3)
-    
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Environment Controls
-        1)
-        2)
-        3)
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Infotainment System
-        1)
-        2)
-        3)
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1)
-        2)
-        3)
-    Name 2 things you can do with the:   Seat
-        1)
-        2)
+1)  Fill in #1 - 4 with a random UDT in plain english
+    These 4 UDTs do not need to be related.
+        a) For each plain-english UDT, write out 5 traits or properties and 3 things it can do
+        b) remember: these properties will eventually become primitives.
+        pick properties that can be represented with 'int float double bool char std::string'
 */
+
+/*
+ example:  
+ 
+Thing: Car Wash   
+    5 properties:
+        - number of vacuum cleaners
+        - number of eco-friendly cleaning supplies
+        - the amount of water used per week.
+        - amount of profit made per week
+        - number of cars serviced per day
+    3 things it can do:
+        - wash and wax car
+        - charge customer
+        - detail the car interior
+
+    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
+    
+    These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+
+    In addition to the usual primitives (covered in Project 2), you can use 'std::string' to represent strings in this project.
+ */
+
+/*
+Thing 1) GYM
+5 properties:
+    1) Number of Exercise Machines
+    2) Number of Weekly Customers
+    3) Average time people spend at GYM
+    4) Monthly cost to customer
+    5) amount of profit
+3 things it can do:
+    1)provide excersise equipment 
+    2)charge customers
+    3)amount of profit
+ */
+
+/*
+Thing 2) Games Console
+5 properties: 
+    1) Amount of storage memory
+    2) Amount of downloaded games
+    3) Turn on/off button
+    4) HDMI Input and output (y/n)
+    5) Amount of usb slots
+3 things it can do:
+    1)Connect to the internet
+    2)Play games from a disk
+    3)Turn on
+ */
+
+/*
+Thing 3) Rollercoaster
+5 properties:
+    1) Amount of seats
+    2) number of max speed
+    3) Power on/off controls
```

Comment:
These are objects, not primitives!   You need to pick things that can be represented with `int float double bool char std::string`
- [ ] revise